localisation.lua

```diff
@@ -0,0 +1,44 @@
+-- CHANNELS
+local CHANNELS = {}
+CHANNELS.SAY = "SAY"
+CHANNELS.RAID = "RAID"
+CHANNELS.WHISPER = "WHISPER"
+CHANNELS.CUSTOM = "CHANNEL"
+CHANNELS.BATTLEGROUND = "INSTANCE_CHAT"
+BuffDuty.CHANNELS = CHANNELS
+
+local SUPPORTED_CHANNELS = {
+    -- English
+    ["SAY"] = CHANNELS.SAY,
+    ["S"] = CHANNELS.SAY,
+    ["RAID"] = CHANNELS.RAID,
+    ["R"] = CHANNELS.RAID,
+    ["WHISPER"] = CHANNELS.WHISPER,
+    ["W"] = CHANNELS.WHISPER,
+    ["CHANNEL"] = CHANNELS.CUSTOM,
+    ["C"] = CHANNELS.CUSTOM,
+    ["CUSTOM"] = CHANNELS.CUSTOM,
+}
+BuffDuty.SUPPORTED_CHANNELS = SUPPORTED_CHANNELS
+
+-- CLASSES
+local CLASSES = {}
+-- English
+CLASSES.MAGE = "MAGE"
+CLASSES.PRIEST = "PRIEST"
+CLASSES.DRUID = "DRUID"
+CLASSES.PALADIN = "PALADIN"
+BuffDuty.CLASSES = CLASSES
+
+local SUPPORTED_CLASSES = { 
+    -- English
+    ["MAGE"] = CLASSES.MAGE,
+    ["M"] = CLASSES.MAGE,
+    ["PRIEST"] = CLASSES.PRIEST,
+    ["P"] = CLASSES.PRIEST,
+    ["DRUID"] = CLASSES.DRUID,
+    ["D"] = CLASSES.DRUID,
+    ["PALADIN"] = CLASSES.PALADIN,
+    ["PA"] = CLASSES.PALADIN,
```

Comment:
```suggestion
    ["PA"] = CLASSES.PALADIN,
    ["HUNTER"] = CLASSES.HUNTER,
    ["H"] = CLASSES.HUNTER,
    ["ROGUE"] = CLASSES.ROGUE,
    ["R"] = CLASSES.ROGUE,
    ["SHAMAN"] = CLASSES.SHAMAN,
    ["S"] = CLASSES.SHAMAN,
    ["WARLOCK"] = CLASSES.WARLOCK,
    ["W"] = CLASSES.WARLOCK,
    ["WARRIOR"] = CLASSES.WARRIOR,
    ["WARR"] = CLASSES.WARRIOR,
```

Reply:
I'm wondering if we want to depricate the short class names... it works OK for most classes, but Warrior and Paladin are a bit, hmm. And it's not like it saves that many characters.

Reply:
Well, that would be a breaking change. If it's not too much trouble let's leave it until we implement UI.